skylib/package_names.bzl

```diff
@@ -229,6 +235,13 @@ def _jessie_java_no_ca_certs_package_names():
 def _jessie_java_package_names():
     return _jessie_java_no_ca_certs_package_names() + ["ca-certificates-java"]
 
+def _jessie_less_package_names():
+    return [
+        "libc6",
+        "libtinfo5",
+        "debianutils",
```

Comment:
this should probably include the "less" package?

Reply:
Thanks! I am still doing testing (that's why I haven't requested review), and I just found the issue you indicated!